charts/hono/Chart.yaml

```diff
@@ -15,7 +15,7 @@ name: hono
 description: |
   Eclipse Hono™ provides remote service interfaces for connecting large numbers of IoT devices to a back end and
   interacting with them in a uniform way regardless of the device communication protocol.
-version: 1.4.9
+version: 1.4.10
 # Version of Hono being deployed by the chart
 appVersion: 1.4.3
```

Comment:
There is no JDBC device registry docker image in the Docker Hub for this version. As you already mentioned in the PR description, we have to wait until the next release and also the app version should be updated accordingly.

Reply:
appVersion changed to 1.5.0